### PR TITLE
clarifying when to add pkgs for migration

### PR DESCRIPTION
**Pull request by ngam**
as discussed quickly with @isuruf, one should NOT add packages like cypthon, numpy, etc. if they are not used in host.

## how to add a blog post

1. Put your post in `.rst` format in the `posts` directory. Look at the other posts for the format. 
   Make sure you have a `.. post::` directive at the top and a section title at the top.
2. You can build your post via the commands `make clean`, `ablog build`, and `ablog serve`.
3. Checkout the results in your browser.
4. When you are ready, make a PR!

### CODE REVIEW

**Comment by ngam** [comment]
cc @isuruf

File: posts/2020-10-29-macos-arm64.rst [hunk 1]
```diff
@@ -123,7 +123,7 @@ You can rerender using,
 
    conda smithy rerender
 
-For python packages, add the following to ``recipe/meta.yaml``,
+For python packages, add one or more of the following to ``recipe/meta.yaml`` if they are used in `host:`,
```
Comment (isuruf) [inline posts/2020-10-29-macos-arm64.rst]: This should only apply to cython, numpy and pybind11

Comment (ngam) [reply posts/2020-10-29-macos-arm64.rst]: Better now?

For python packages, add one or more of the following to ``recipe/meta.yaml`` as needed, noting that you *must* only add `numpy`, `cython`, and/or `pybind11` if  they are used in `host:` as well,

**Comment by isuruf** [comment]
Thanks

**Comment by ngam** [comment]
Thank YOU!

**Pull request merged**